04_numeronym/hsy3418/main_test.go

```diff
@@ -0,0 +1,33 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+)
+
+func TestNumeronym(t *testing.T) {
+
+	actual := numeronyms("accessibility", "Kubernetes", "abc", "Android Enterprise", "", "ab")
+	expected := []string{"a11y", "K8s", "a1c", "A16e", "", "ab"}
+	for index, val := range actual {
+		if expected[index] != val {
+			t.Errorf("Unexpected output in main()\nexpected: %q\nactual: %q", expected[index], actual)
+		}
+	}
+
+}
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := strconv.Quote("[a11y K8s a1c]\n")
```

Comment:
Sorry I didn't notice this before but... the requirement is that 'abc' should return as 'abc' (not 'a1c').  Anything three chars or less should return as per how it was received..